src/modules/commander/commander.cpp

```diff
@@ -2881,7 +2881,7 @@ int commander_thread_main(int argc, char *argv[])
 						       _mission_result.stay_in_failsafe,
 						       &status_flags,
 						       land_detector.landed,
-						       (rc_loss_enabled > 0),
+						       rc_loss_mode,
```

Comment:
@anton-matosov I think you got it right, that's very much the way it was designed. If you're concerned about breaking the state machine helper the way to resolve that would be to add tests to and then extend or refactor it.